portals/application/languages/en_US/en_US.po

```diff
@@ -2414,8 +2414,8 @@ msgstr ""
 
 msgid ""
 "You can optionally prepend the terminal model or any other subpath to the "
-"url before the MAC address for verbosity/debugging purposes. Only the "
-"filename is checked in specific provisioning."
+"url before the MAC address for verbosity/debugging purposes. File extension "
+"will be ignored if none is set."
```

Comment:
If a extension if given (e.g. {mac}.cfg), requested file must have that extension too. Otherwise, only requested filename will be checked.

Reply:
Applied